plugins/providers/docker/driver.rb

```diff
@@ -151,6 +151,17 @@ def docker_bridge_ip
         end
       end
 
+      def exec(params, **opts, &block)
```

Comment:
We are moving away from keyword args and using `opts = {}` instead. The keyword args require a specific version of Ruby and only work for symbols.